Gemfile

```diff
@@ -1,5 +1,7 @@
 source 'https://rubygems.org'
 
+gem 'react_on_rails', path: '../react_on_rails'
```

Comment:
@justin808 yes we should, that way if you want to view results for something that isn't yet published, you can